src/SFA.DAS.Commitments.Database/PostDeployment/PostDeploymentScript.sql

```diff
@@ -44,4 +44,19 @@ BEGIN
 	 WHERE	a.[Id]				= h.[EntityId]
 	 AND	a.[PaymentStatus]	= 2		  
 
-END
\ No newline at end of file
+END
+
+
+------ Extended EDIT
+
+-- Marking Apprenticeships as HasHadDataLockSuccess so that they cannot be used in extended edit. 
+-- Only if they have a DataLockStatus with ErrirCode Success (0)
+
+UPDATE Apprenticeship 
```

Comment:
This approach will not cater for apprenticeships (/price episodes) that have had a success that has subsequently been superceded by a failure. However, there is no means to write a post deployment script to do this since the data is not available. Therefore we'd have to ask payments for a list and run something manually after deployment. In any case, the behaviour in this scenario would be as now - they wouldn't be able to edit, and would have to stop/restart. So this seems ok to me - thoughts @leewadhams ?

Reply:
Would probably be a small percentage where they've submitted a valid price episode and then submitted another ILR with a failure.